docs/medical-api/api-reference/patient/get-patient-consolidated.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: "Get Patient's Consolidated Data"
+description: "Gets a Patient's Consolidated Data."
+api: "GET /medical/v1/patient/{id}/consolidated"
+---
+
+## Path Params
+
+<ParamField path="id" type="string" required>
+  The ID of the Patient.
+</ParamField>
+
+## Query Params
+
+<ParamField query="resources" type="string">
+  List of FHIR resources to be returned with the patient
+</ParamField>
+
+<Info>
+  To view the list of all available FHIR resources, please refer to the [FHIR
+  Resources](/medical-api/fhir/resources) section.
+</Info>
+
+## Response
+
+Returns a patient as you can see when fetching a [patient](/medical-api/api-reference/patient/get-patient) along with the patient's specified [FHIR resources](/medical-api/fhir/resources).
```

Comment:
For the response snippet i wasnt sure if we want to add the same thing that its in get patient and then make a note that it could have other resources or just one example or if this is fine